coderabbittest.py

```
@@ -0,0 +1,56 @@
+#coderabbit tests
+
+# This script calculates a 'special sum' from a list of numbers.
+# It's designed to be simple but provoke discussion during a code review.
+ 
+import os # Unused import - why is this here?
+ 
+_FACTOR = 2.5 # Global variable - is this good practice? What does it do?
+ 
+def do_work(a_list): # Less descriptive function name.
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)